libdnf/base/goal_elements.cpp

```diff
@@ -25,6 +25,29 @@ along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
 namespace libdnf {
 
 
+bool GroupJobSettings::resolve_strict(const libdnf::ConfigMain & cfg_main) {
```

Comment:
[optional] What about to create a template for `GroupJobSettings::resolve_strict(const libdnf::ConfigMain & cfg_main)` and `GoalJobSettings::resolve_strict(const libdnf::ConfigMain & cfg_main)`. It looks like that they are identical.